src/store/modules/sales.js

```diff
@@ -0,0 +1,41 @@
+import { fetchSales, postSale } from "@/services/HttpClientService";
+
+export default {
+  namespaced: true,
+  state: {
+    sales: [],
+    tableColumns: [
+      { key: "seller", title: "Seller" },
+      { key: "buyer", title: "Buyer" },
+      { key: "products", title: "Products", align: "center" },
+      { key: "totalPrice", title: "Total Price" },
+      { key: "totalDiscountedPrice", title: "Discount" },
+      { key: "totalDiscount", title: "Final price" },
```

Comment:
These are still not switched
```suggestion
      { key: "totalDiscountedPrice", title: "Discounted Price" },
      { key: "totalDiscount", title: "Discount" },
```

Reply:
done